### PR TITLE
Data definition for master module.

### PR DESCRIPTION
**Pull request by piharpi**
Oke bang alip

### CODE REVIEW

File: hotel_realta.sql [hunk 1]
```diff
@@ -1,3 +1,107 @@
-CREATE DATABASE Hotel_Realta
+-- membuat database
+USE tempdb;
```
Comment (alv4n96) [inline hotel_realta.sql:2]: delete nih

File: hotel_realta.sql [hunk 2]
```diff
@@ -1,3 +1,107 @@
-CREATE DATABASE Hotel_Realta
```
Comment (hafizmursidd) [inline hotel_realta.sql:1]: gud job boy

**Review by alv4n96 — COMMENTED** [review]
LGTM!

**Review by hafizmursidd — APPROVED** [review]
tingkatkan boy.

**Review by alv4n96 — APPROVED** [review]
LGTM!

File: hotel_realta.sql (hunk 1 above)
Comment (hafizmursidd) [reply hotel_realta.sql:2]: apaan yang di delete cuk?

**Pull request merged**